api/db/seeds/data/organizations-builder.js

```diff
@@ -58,6 +58,14 @@ module.exports = function organizationsBuilder({ databaseBuilder }) {
     rawPassword: 'Pix123',
     cgu: false
   }).id;
+  const user3Id = databaseBuilder.factory.buildUser.withUnencryptedPassword({
```

Comment:
Proposition de nom de commit
[API] Add SchoolingRegistration to RA testing => [API] Add seed for schooling registration

Reply:
Done

Reply:
Done